client/src/components/Draftail/decorators/TooltipEntity.js

```diff
@@ -5,7 +5,10 @@ import { Icon } from 'draftail';
 import Tooltip from '../Tooltip/Tooltip';
 import Portal from '../../Portal/Portal';
 
-const shortenLabel = (label) => {
+/**
+ * Shortens the given label if it goes beyond a predetermined limit.
+ */
+export const shortenLabel = (label) => {
```

Comment:
I think we might want to write a couple of tests for this function now that it’s reused elsewhere – basically test what happens with a label shorter than 25 chars, and longer.